pandas/tools/tile.py

```diff
@@ -185,6 +185,13 @@ def qcut(x, q, labels=None, retbins=False, precision=3, duplicates='raise'):
     x, dtype = _coerce_to_type(x)
 
     if is_integer(q):
+        if x.size == 0:
+            raise ValueError('Cannot qcut empty array')
+
+        rng = (nanops.nanmin(x), nanops.nanmax(x))
+        if rng[0] == rng[1] and q == 1:
```

Comment:
@luca-s I still find this a bit odd, can you do this by simply adjusting the values (and not using this kw, even though its only internal)?